clash-vexriscv-sim/tests/tests.hs

```diff
@@ -104,7 +104,7 @@ runTest ::
   FilePath ->
   TestTree
 runTest name mode n elfPath expectPath =
-  testCase ("Integration test `" <> name <> "` (" <> mode <> ")") $ do
+  testCase ("Integration test " <> name <> " (" <> mode <> ")") $ do
```

Comment:
I did this so I can filter this test out with `-p`. If there are backticks / quotes in there I'd have to use non-obvious escaping (at the very least `\` doesn't work..).